includes/definitions/discovery/alaxala.yaml

```diff
@@ -0,0 +1,19 @@
+modules:
+    mempools:
+        data:
+            -
+                total: AX3630S::ax3630sMemoryTotalSize
+                used:  AX3630S::ax3630sMemoryUsedSize
+                precision: 1024
+    os:
+        hardware:
+            - AX3630S::ax3630sModelType.0

```

Comment:
@NightowlKr I think you are either misunderstanding how SNMP works or how LibreNMS works..

1. By default the sysObjectID match key in yaml matches the START of the sysObjectID variable.
2. The sysObjectID does NOT determine what OID other data returned by the device lives in the OID tree.  For example, Devices can return data in a completely different address space.  Vendors should NOT make a mib per device and copy MIB under that OID, this is VERY bad.
3. 21839 is the enterprise id assigned to this vendor by IANA, any private MIB data should live under that OID.
4. This means if we set `sysObjectID: .1.3.6.1.4.1.21839.` It will match every device made by this vendor (assuming they follow industry established conventions)
5. I checked the stats site, currently, no one currently has devices under this vendor's enterprise id reported.

Reply:
@murrant Thank you for the additional explanation.

I've known about the SNMP OID regulations.
The reason I tried to limit the sysObjectID is to apply PHP Discovery to other devices.

Because when trying to the PHP discovery test code, earlier registered devices process and memory data  isn't collected.